docs/docker-image-details.md

```diff
@@ -68,6 +68,10 @@ You can use this kind of images to test the catalog of Apache Hive.
 
 Changelog
 
+- gravitino-ci-hive:0.1.7
+  - Download Mysql JDBC driver before building the Docker image
+  - Set `hdfs` as HDFS superuser group
+
 - gravitino-ci-hive:0.1.6
   - No start YARN when container startup
```

Comment:
Could you modify this grammar by the way?
`No start` -> `No starting`

Reply:
OK, fixed